### DISCUSSION TITLE
Test easy-1 parte 2

### DISCUSSION THREAD

**Discussion by ncruzm** [IIC2133-PUC/2024-2#170] [Tarea 3] [answered]
Hola en el test easy-1 de la parte 2, veo que en las ultimas lineas del input.txt (adjunto imagen) se repite una arista y con diferente costo, sera un error? O es parte del problema?
![Captura de pantalla 2024-11-09 a las 10 46 31](https://github.com/user-attachments/assets/7a06c8fc-fdb2-4690-b819-c2b2aab13522)

**Comment by ncruzm** [comment]
Tambien se supone que los inputs del enunciado serian de la forma
N
L
Lineas....

Pero veo que N y L van en la misma linea

**Comment by Segismundios** [comment, marked as answer]
Holaa @ncruzm. Con relación a tu primera pregunta, sí puede haber más de una arista entre dos nodos. 
En cuanto a los tests, ya se encuentran actualizados al formato indicado en el enunciado.
Saludos!
